python/ctsm/crop_calendars/check_rx_obeyed.py

```diff
@@ -104,6 +104,27 @@ def get_extreme_info(diff_array, rx_array, mxn, dims, gs_da, patches1d_lon, patc
     return round(themxn, 3), round(this_lon, 3), round(this_lat, 3), this_gs, round(this_rx)
 
 
+def summarize_results(which_ds, output_var, verbose, all_ok, gdd_tolerance, diffs_eg_txt):
+    """
+    Summarize results
+    """
+    bad = True
+    if all_ok == 2:
+        bad = False
+        print(f"✅ {which_ds}: Prescribed {output_var} always obeyed")
```

Comment:
I realize this was from before -- but I love the special characters here, with the green checkbox, yellow box and red X. That makes it easy to see what's going on here. This might be something we should use elsewhere. Is this something you read about or saw elsewhere, or just figured out on your own?

Reply:
It's something I made up myself; I use emoji all over my notes and figured it'd be nice to do the same here.

Reply:
Cool, I'll keep this idea in mind for the future.